aspnetcore/signalr/java-client.md

```diff
@@ -0,0 +1,68 @@
+---
+title: ASP.NET Core SignalR Java client
+author: mikaelm12
+description: Learn how to use the ASP.NET Core SignalR Java client.
+monikerRange: '>= aspnetcore-2.2'
+ms.author: mimengis
+ms.custom: mvc
+ms.date: 08/30/2018
+uid: signalr/java-client
+---
+# ASP.NET Core SignalR Java Client
+
+By [Mikael Mengistu](https://twitter.com/MikaelM_12)
+
+The Java client enables connecting to an ASP.NET Core SignalR server from Java code, including Android apps. Like the [JavaScript client](xref:signalr/javascript-client) and the [.NET client](xref:signalr/dotnet-client), the Java client enables you to receive and send messages to a hub in real time. The Java client is available in ASP.NET Core 2.2 and later.
+
+The sample Java console app referenced in this article uses the Java client.
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/signalr/java-client/sample) ([how to download](xref:tutorials/index#how-to-download-a-sample))
+
+## Install the SignalR Java client package
+
+The *signalr-0.1.0-preview1-35029* JAR file allows clients to connect to SignalR hubs.
+
+If using Gradle, add the following line to the `dependencies` section of your *build.gradle* file:
+
+```gradle
+implementation 'com.microsoft.aspnet:signalr:0.1.0-preview1-35029'
+```
+
+If using Maven, add the following lines inside the `<dependencies>` element of your *pom.xml* file:
+
+[!code-xml[pom.xml dependency element](java-client/sample/pom.xml?name=snippet_dependencyElement)]
+
+## Connect to a hub
+
+To establish a connection, create a `HubConnectionBuilder` and call `build`. The hub URL and log level can be configured while building a connection. Configure any required options by inserting any of the `HubConnectionBuilder` methods into `build`. Start the connection with `start`.
```

Comment:
We might want to add a note to use regular `HubConnection` constructor for Android applications. I'll get back to you on this.